.env.example

```diff
@@ -0,0 +1 @@
+ALLOWED_HOSTS=...,...,...
```

Comment:
probably no need to merge this as a separate file to main development branch, maybe it could be a line of comment above code that reads from .env in settings.py?